deploy/roles/devtools6/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install devtools repo
+  template:
+    src: devtools-6.repo.j2
+    dest: /etc/yum.repos.d/devtools-6.repo
+    mode: 0644
+  when: not custom_repo
+  tags:
+    - devtools
+
+- name: Install updated devtools
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - devtoolset-6-binutils
+    - devtoolset-6-build
+    - devtoolset-6-gcc
+    - devtoolset-6-gcc-c++
+    - devtoolset-6-gcc-gfortran
+    - devtoolset-6-gdb
+    # - devtoolset-6-git
```

Comment:
Doesn't appear to be, unless I'm not looking in the right places. I couldn't find any information regarding the removal.